src/ui/autocomplete.ts

```diff
@@ -442,6 +457,28 @@ class AutocompleteUI {
       onSelection(result);
     }
 
+    // Radar Autocomplete Session
+    if (this.session) {
+      const userId = Storage.getItem(Storage.USER_ID) || undefined;
```

Comment:
Right now I'm still including these even though right now we don't use them, but in case we wanted to store them in the future so we don't have to update the sdk twice